percona/watcher/wal.go

```diff
@@ -167,6 +167,7 @@ func GetLatestCommitTimestamp(ctx context.Context, cli client.Client, execCli *c
 
 	primary, err := perconaPG.GetPrimaryPod(ctx, cli, cr)
 	if err != nil {
+		log.Error(err, "failed to get primary pod")
```

Comment:
In the Jira ticket description, it is suggesting the wrapping of the error like that

```
errors.Wrap(err, PrimaryPodNotFound.Error())   
```

Why aren't we wrapping the error, and instead we are adding logs?

Reply:
fixed!